walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +273,155 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void ignoreOptionalAttendeeSchedule() {
+    // Have on optional attendee, Person C, who has an all-day event.
+    // Have each mandatory person have different events. We should see three options because each person has
+    // split the restricted times and the optional attendee's schedule should not be considered. 
+    //           |--------------C--------------|
```

Comment:
Nice use of diagrams! This really helps make the test case clear.

Reply:
Thanks!